src/basic/namespace-util.c

```diff
@@ -34,6 +34,14 @@ const struct namespace_info namespace_info[] = {
 
 #define pid_namespace_path(pid, type) procfs_file_alloca(pid, namespace_info[type].proc_path)
 
+static NamespaceType clone_flag_to_namespace_type(unsigned long clone_flag) {
+        for (NamespaceType t = 0; t < _NAMESPACE_TYPE_MAX; t++)
```

Comment:
FOREACH_ARRAY?

Reply:
doesn#t really work, we actually want to return `t` here once we found what we are looking for.

i mean, we could calculate it from the pointers again but not sure that#s prettier...